File: tests/fixtures/cpp/sample.stripped.cpp
namespace calc {

class Calculator {
    int total = 0;
    static const std::vector<int> kSeeds;

public:
    explicit Calculator(int seed) : total(seed) {}

    int add(int a, int b) {}

    int total_value() const {}
};

const std::vector<int> Calculator::kSeeds = {};

auto square = [](int x) {};

}  

File: tests/fixtures/java/sample.stripped.java
package com.example.calc;


public abstract class Calculator {
    private int total;
    static final String BANNER = "Calculator";

    public Calculator(int seed) {}

    public int add(int a, int b) {}

    public int apply(IntBinaryOperator op, int a, int b) {}

    public abstract int describe();
}

